### DISCUSSION TITLE
Implement A Distance Scale When In Navigation Mode

### DISCUSSION THREAD

**Discussion by Scrapporoid** [organicmaps/organicmaps#2860] [Ideas]
When in non-navigation mode, a distance scale is provided at the lower left corner of the screen.  However, when in navigation mode, there is no distance scale.  Adding one (especially tick-marked up one side or even better along the path of travel) would be helpful, IMO.  

As a former military navigator, range rings would be cool, but I won't ask for that. ;)

Thanks for your consideration of this.

Regards,
Scrapporoid

**Comment by biodranik** [comment]
It's a good idea. How other navigators are implementing it? Any screenshots?

**Reply by Scrapporoid** [reply]
Apologies for only just now noticing your reply, biodranik!  Here's an example of an implementation, in a tool called [JBV1](https://play.google.com/store/apps/details?id=com.johnboysoftware.jbv1), which integrates multi-source data with output from your own Valentine1 radar detector.  In this instance, the outer limit of the display is the 1-mile range ring, and you can see the 1/2- and 1/4-mile rings as well.  For what it's worth, this is the only map-related app I leave in auto-scaling, as speed is relevant to the police radar threats; in this case, I have it set to scale to 1.5 miles at 45 mph and 2.0 miles at 60 mph.

![_temp_1](https://user-images.githubusercontent.com/97354126/216427914-8685658e-0f29-4659-baaf-ac1efc207aaa.png)
